scripts/generate-schematic-imports-map.ts

```diff
@@ -0,0 +1,53 @@
+import {sync as glob} from 'glob';
```

Comment:
I'm not sure whether we want to check it in. Since we do not expect to have such a migration in the future anymore, we can just remove it from the repo, and if we ever need it again, either build it again (as it seems quite short) or just pull it from a `gist.github.com`.

I guess it's also not a deal to keep it in the repo 😄 It's just a thing more we need to delete (if we know when we even can delete it)

Reply:
Eh, it could come in handy at some point. We can always delete it later